src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
+    @Column(unique = true)
```

Comment:
Why do we need password to be unique?

Reply:
Yea, it`s a mistake, annotation shuld be above email.